app/Enums/AvailableStore.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Enums;
+
+enum AvailableStore: string
```

Comment:
Jeśli korzystamy z Adapterów to tutaj powinny być zwracane na podstawie adapterów, gdyby ktoś chciał dodać swój adapter to dodanie go do configa nie wystarczy. Ja wiem że to jest prostrze ale spróbuj to jakiej domeny dotyczy adapter przenieść do samego adaptera :)